app/features/feedback/classifier/helpers/get-feedback-messages.spec.js

```diff
@@ -0,0 +1,68 @@
+import { expect } from 'chai';
+import { getFeedbackMessages } from './open-feedback-modal';
+
+const MULTI_TRUE_MULTI_FALSE = [
+  true,
+  true,
+  true,
+  false,
+  false
+];
+
+const MULTI_TRUE_SINGLE_FALSE = [
+  true,
+  true,
+  true,
+  false
+];
+
+const SINGLE_TRUE_MULTI_FALSE = [
+  true,
+  false,
+  false,
+  false
+];
+
+function generateMinimalRule(status, ruleID) {
+  return {
+    id: ruleID,
+    failureEnabled: true,
+    successEnabled: true,
+    failureMessage: 'miss',
+    successMessage: 'hit',
+    success: status
+  };
+}
+
+const MULTI_TRUE_MULTI_FALSE_RULES = MULTI_TRUE_MULTI_FALSE.map(generateMinimalRule);
+const MULTI_TRUE_SINGLE_FALSE_RULES = MULTI_TRUE_SINGLE_FALSE.map(generateMinimalRule);
+const SINGLE_TRUE_MULTI_FALSE_RULES = SINGLE_TRUE_MULTI_FALSE.map(generateMinimalRule);
+
+describe('helpers > open-feedback-modal > getFeedbackMessages', function() {
+  it('should exist', function() {
+    expect(getFeedbackMessages).to.be.a('function')
+  })
+
+  it('should reduce an array of messages to a collection of unique messages with counts. ' +
```

Comment:
Just an organizational suggestion, but you can put this repeated message into another describe block (describe blocks can be nested), so:
```
describe('when reducing an array of messages to a collection of unique messages with counts', function ()
 {
   it('expectation description', function () {

   })
})
```